appveyor.yml

```diff
@@ -11,11 +11,11 @@ environment:
 
     matrix:
         - PYTHON: C:\Python27
-          TOX_ENV: py27-test
+          TOX_ENV: py27-testwin32
         - PYTHON: C:\Python34
-          TOX_ENV: py34-test
+          TOX_ENV: py34-testwin32
         - PYTHON: C:\Python35
-          TOX_ENV: py35-test
```

Comment:
Ah, I see now—you're using this flag to disable tests on the command line.

Instead would you mind looking into using `@unittest.skipIf()` or `self.skipTest()` in the tests themselves? This makes for a graceful degradation: developers can just type `nosetests`, regardless of whether they're on Windows or not, and have the appropriate set of tests skipped based on their current configuration. There are a few examples of this in the current test suite in case that's helpful.